srcpkgs/radicale/files/radicale/log/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
log services should be created automatically by xbps-src if the directory doesn't exist in the service in files/